src/app/cli/src/app.rs

```diff
@@ -53,13 +46,20 @@ pub async fn run(
         std::env::set_var("RUST_BACKTRACE", "1");
     }
 
-    let workspace_svc = WorkspaceService::new(Arc::new(workspace_layout.clone()));
+    // Relevant for cases where we already have a config, but no workplace yet
+    // ("kamu init" call)
```

Comment:
Why would we need the database in this scenario?

Reply:
The workspace might not exist and this might not be the workspace creation command
There migth be still a global configuration user's home foder, I suppose, and it can be loaded.

Reply:
> Why would we need the database in this scenario?

this change does not apply to the database

---

I fixed the `kamu init --multi-tenant` command in the case where a config with predefined users is present. 

Otherwise, I was getting the following assert triggered:

```rust
pub fn register_config_in_catalog(
    config: &config::CLIConfig,
    catalog_builder: &mut CatalogBuilder,
    multi_tenant_workspace: bool,
) {
    // ...
    if multi_tenant_workspace {
        // ...
    } else {
        if let Some(users) = &config.users {
            assert!(
                users.predefined.is_empty(),
                "There cannot be predefined users in a single-tenant workspace"
            );
        }
        // ...
    }
}
```

Reply:
Since the comment left room for interpretations and questions, then it is not accurate. For this reason, I have updated it with more context